lib/mayaUsd/render/vp2ShaderFragments/FallbackCPVShaderStandardSurface.xml

```diff
@@ -0,0 +1,130 @@
+        <float3  name="Pw" ref="mayaDefaultStandardSurface.Pw" semantic="Pw" />
+        <float4x4  name="ViewProj" ref="mayaDefaultStandardSurface.ViewProj" semantic="viewprojection" />
+        <float  name="fogStart" ref="mayaDefaultStandardSurface.fogStart" />
+        <float  name="fogEnd" ref="mayaDefaultStandardSurface.fogEnd" />
+        <int  name="fogMode" ref="mayaDefaultStandardSurface.fogMode" />
+        <float  name="fogDensity" ref="mayaDefaultStandardSurface.fogDensity" />
+        <float4  name="fogColor" ref="mayaDefaultStandardSurface.fogColor" />
+        <float  name="fogMultiplier" ref="mayaDefaultStandardSurface.fogMultiplier" />
+        <float3  name="IrradianceEnv" ref="mayaDefaultStandardSurface.IrradianceEnv" />
+        <float3  name="SpecularEnv" ref="mayaDefaultStandardSurface.SpecularEnv" />
+        <int  name="selectionIndex" ref="mayaDefaultStandardSurface.selectionIndex" />
+        <struct  name="light0" ref="mayaDefaultStandardSurface.light0" />
+        <struct  name="light1" ref="mayaDefaultStandardSurface.light1" />
+        <struct  name="light2" ref="mayaDefaultStandardSurface.light2" />
+        <struct  name="light3" ref="mayaDefaultStandardSurface.light3" />
+        <struct  name="light4" ref="mayaDefaultStandardSurface.light4" />
+        <struct  name="light5" ref="mayaDefaultStandardSurface.light5" />
+        <struct  name="light6" ref="mayaDefaultStandardSurface.light6" />
+        <struct  name="light7" ref="mayaDefaultStandardSurface.light7" />
+        <struct  name="light8" ref="mayaDefaultStandardSurface.light8" />
+        <struct  name="light9" ref="mayaDefaultStandardSurface.light9" />
+        <struct  name="light10" ref="mayaDefaultStandardSurface.light10" />
+        <struct  name="light11" ref="mayaDefaultStandardSurface.light11" />
+        <struct  name="light12" ref="mayaDefaultStandardSurface.light12" />
+        <struct  name="light13" ref="mayaDefaultStandardSurface.light13" />
+        <struct  name="light14" ref="mayaDefaultStandardSurface.light14" />
+        <struct  name="light15" ref="mayaDefaultStandardSurface.light15" />
+        <float3  name="Nw" ref="mayaDefaultStandardSurface.N" flags="varyingInputParam" />
+        <float3  name="Vw" ref="mayaDefaultStandardSurface.V" flags="varyingInputParam" />
+        <string  name="selector" ref="mayaDefaultStandardSurface.selector" />
+        <float3  name="ambientColor" ref="mayaDefaultStandardSurface.ambientColor" />
+        <float3  name="ambientIn" ref="mayaDefaultStandardSurface.input" />
+        <undefined  name="GPUStage" ref="mayaDefaultStandardSurface.GPUStage" semantic="GPUStage" />
+        <float4 name="diffuseColor" ref="mayaCPVPassing.colorIn" flags="varyingInputParam" />
+    </properties>
+    <values>
+        <float name="base" value="1.000000"  />
+        <float name="specular" value="1.000000"  />
+        <float3 name="specularColor" value="1.000000,1.000000,1.000000"  />
```

Comment:
If you want to aim for a Lambert flat look for the default material I would recommend setting the `specular` parameter to zero and the `diffuseRoughness` parameter to 1.0.

Reply:
I wasn't aiming for a flat look but replicating the look of 'standardSurface1' or the new default, which does have spec to it at the values I have set there. I'm happy to implement as a flat material as well though, your call.

Reply:
Matching standardSurface1 works for me. I see you already have the correct specularRoughness selected.